tests/FakeItEasy.Tests.Approval/ApiApproval.cs

```diff
@@ -52,6 +52,30 @@ public void ApproveApiNetStd21()
             ApproveApi("netstandard2.1");
         }
 
+        [Fact]
+        [UseReporter(typeof(DiffReporter))]
+        [MethodImpl(MethodImplOptions.NoInlining)]
+        public void ApproveExtensionsValueTaskApi45()
+        {
+            ApproveExtensionsValueTaskApi("net45");
+        }
+
+        [Fact]
+        [UseReporter(typeof(DiffReporter))]
+        [MethodImpl(MethodImplOptions.NoInlining)]
+        public void ApproveExtensionsValueTaskApiNetStd16()
+        {
+            ApproveExtensionsValueTaskApi("netstandard1.6");
+        }
+
+        [Fact]
+        [UseReporter(typeof(DiffReporter))]
+        [MethodImpl(MethodImplOptions.NoInlining)]
+        public void ApproveExtensionsValueTaskApiNetStd20()
+        {
+            ApproveExtensionsValueTaskApi("netstandard2.0");
+        }

```

Comment:
I did. But ApprovalTests relies on the name of the test, so I'm not sure it would work with a theory...

Reply:
Worth a try, though!

Reply:
> But ApprovalTests relies on the name of the test,

I did not know this. Or had forgotten!

Reply:
I'll give it a try anyway. If it doesn't work, at least the `ApproveApi` method can probably be refactored to accept the project name, instead of duplicating it.

Reply:
Got it working! ApprovalTests does *not* make it easy to control the approved/received file names... I spent at least half an hour reverse engineering it to understand how to achieve this.

I'll send a separate PR for this. This one is already big enough, and doesn't need the additional confusion.